src/modules/container.rs

```diff
@@ -12,6 +12,16 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     use crate::formatter::StringFormatter;
     use crate::utils::read_file;
 
+    fn check_if_docker_in_cgroup(context: &Context) -> Option<String> {
+        Some(
+            context
+                .exec_cmd("grep", &["-cim1", "/docker", "/proc/1/cgroup"])?
+                .stdout
+                .trim()
+                .to_string(),
```

Comment:
The method seems fine, but please handle this with native rust code (`File::open`, `BufReader`, `.lines()` and perhaps `regex`).

Reply:
Done